server/api/admin.js

```diff
@@ -0,0 +1,111 @@
+const router = require('express').Router()
+module.exports = router
+
+const {User, Song} = require('../db/models')
+
+// api/admin returns all user data
+router.get('/', async (req, res, next) => {
+  const isAdmin = req.user.role === 'admin'
+
+  try {
+    if (isAdmin) {
+      const allUserData = await User.findAll()
+      res.send(allUserData)
+    } else {
+      res.send(404)
+    }
+  } catch (error) {
+    next(error)
+  }
+})
+
+// api/admin/create/song enables admin to modify song data song data
```

Comment:
Typo on line 22 and I don't think we need lines 23-32

Reply:
@camitta lines 23-32 describe what JSON schema the endpoint accepts, since we don't have a place to document that, I left them in the comments